deploy/sre-pruning/osd-15020-rpo-412/115-pruning-servicemonitors.CronJob.yaml

```
@@ -0,0 +1,37 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: services-pruner
+  namespace: openshift-sre-pruning
+spec:
+  failedJobsHistoryLimit: 5
+  successfulJobsHistoryLimit: 3
```

We probably don't need this many
```suggestion
  failedJobsHistoryLimit: 3
  successfulJobsHistoryLimit: 1
```